### ISSUE TITLE
Documentation build error 

### ISSUE THREAD

**Issue by yebai** [TuringLang/turing.ml#27]
It seems the docs build is broken by some recent updates to `TuringTutorials`. 

```julia
Run julia --project=docs --color=yes make.jl
  Activating environment at `~/work/turing.ml/turing.ml/Project.toml`
     Cloning git-repo `[https://github.com/TuringLang/TuringTutorials`](https://github.com/TuringLang/TuringTutorials%60)
    Updating git-repo `[https://github.com/TuringLang/TuringTutorials`](https://github.com/TuringLang/TuringTutorials%60)
ERROR: LoadError: LoadError: Did not find rev artifacts in repository
Stacktrace:
```

https://github.com/TuringLang/turing.ml/runs/6032087065?check_suite_focus=true#step:6:11

cc @devmotion @rikhuijzer

**Comment by devmotion** [comment]
Yes, we noticed in https://github.com/TuringLang/turing.ml/pull/26 that the branch was removed. I don't know when this happened since it is unrelated to the recent changes.

I didn't dare to merge https://github.com/TuringLang/turing.ml/pull/26 since I'm too unfamiliar with the build setup here but I assume it should work and use the updated tutorials (and hence fix the error).

**Comment by devmotion** [comment]
Should be fixed by #26, the docs use TuringTutorialsOutputs now instead of TuringTutorials.
